src/channel/opts.js

```diff
@@ -114,10 +114,17 @@ OptionsModule.prototype.handleSetOptions = function (user, data) {
     if ("voteskip_ratio" in data) {
         var ratio = parseFloat(data.voteskip_ratio);
         if (isNaN(ratio) || ratio < 0) {
-            ratio = 0;
+            user.socket.emit("validationError", {
+                target: "#cs-voteskip_ratio",
+                message: "Input must be a number 0 or greater. 1.0 is 100%."
```

Comment:
Just a nitpick, but I like to get in the habit of including the erroneous value just to make it more obvious when something is wrong.  For example:

```
message: `Input must be a number 0 or greater, not "${data.voteskip_ratio}"`
```

Other than that, it looks fine to me.